core/services/relay/evm/codec_entry.go

```diff
@@ -50,6 +50,8 @@ func (entry *codecEntry) Init() error {
 		}
 		tag := reflect.StructTag(`json:"` + arg.Name + `"`)
 		name := strings.ToUpper(arg.Name[:1]) + arg.Name[1:]
+		// this allows us to decode "foo" as "Foo", since go requires exported fields to start with a capital letter
```

Comment:
this comment should be above `name := strings.ToUpper(arg.Name[:1]) + arg.Name[1:]`

Reply:
I meant to revert this file.